nservicebus/azure-service-bus/performance-tuning_body_asb_[7,).partial.md

```diff
@@ -0,0 +1,63 @@
+## Tuning send performance outside the scope of a handler
+
+The following ASB SDK settings impact sending operations outside of the handler context
+
+  * `MessagingFactories().BatchFlushInterval()`: Controls the time that the ASB SDK batches client side requests before sending them out. Optimal values are between 100 and 500 milliseconds. To turn this feature off, provide TimeSpan.Zero.
+  * `MessagingFactories().NumberOfMessagingFactoriesPerNamespace()`: Each factory manages a TCP connection to a front end node in the broker, this connection has throughput limits. Opening more factories significantly improves send performance. Optimal values are between 2 and 8 for sending.
+  * `NumberOfClientsPerEntity()`: Keep this value equal to the number of factories, this ensures there is 1 send client per factory.
+
+The way the send API is invoked matters a lot as well. Using `await` on each `Send()` operation will make the ASB SDK wait until the send operation completes and therefore makes client side batching useless. It's better to maintain a list of send tasks and await them all together instead.
+
+Slow:
+
+Snippet: asb-slow-send
+
+Fast:
+
+Snippet: asb-fast-send
+
+## Tuning receive performance
+
+Tuning receive performance depends on quite a few things that vary per scenario, but the following settings bring improvements for all scenarios.
+
+  * `MessagingFactories().NumberOfMessagingFactoriesPerNamespace()`: Each factory manages a TCP connection to a front end node in the broker, this connection has throughput limits. Opening more factories significantly improves receive performance. Optimal value ares between 16 and 40 for receiving.
+  * `NumberOfClientsPerEntity()`: Keep this value equal to the number of factories, this ensures there is 1 internal receive client per factory.
+
+Other settings that impact performance a lot, but their values differ quite a bit per scenario, are:
```

Comment:
I'll go for 'Changing the following settings will also improve application performance, but their values depend on the nature of the application'